skyportal/utils/offset.py

```diff
@@ -83,6 +83,9 @@ def _connect(self):
         except (HTTPError, ConnectionResetError):
             log("Warning: main Gaia TAP+ server failed")
             self.is_backup = True
+        except:  # noqa: E722
+            log("Warning: main Gaia TAP+ server failed in a way we didn't expect.")
```

Comment:
Log the exception here, so we can see what type it is and what error message is being returned.

Reply:
Done.